configuration/yaml.md

```diff
@@ -0,0 +1,338 @@
+      <channel_id>:
+        type: <channel type>
+        kind: state|trigger # Default: state
+        itemType: <item type>
+        itemDimension: <item dimension> # Only if itemType is "Number"
+        label: <channel label>
+        description: <description>
+        config:
+          <channel config key>: <config value>
+```
+
+| Configuration Key | Description                                                                                                                                     |
+|:------------------|:------------------------------------------------------------------------------------------------------------------------------------------------|
+| `<thing_uid>`     | The UID of the Thing or Bridge being defined. The UID format is `<binding_id>:<thing_type_id>:[<bridge_id>:]<thing_id>`, e.g. `astro:sun:home`. |
+| `isBridge`        | Whether the entity is a Bridge (`true`) or a Thing (`false`). Default: `false` (a Thing).                                                       |
+| `bridge`          | The bridge UID that this Thing belongs to.                                                                                                      |
+| `label`           | Thing label.                                                                                                                                    |
+| `location`        | The location of the Thing.                                                                                                                      |
+| `config`          | A key-value map of the Thing's configuration. Refer to the Binding's documentation for details.                                                 |
+| `channels`        | A map of custom channels. Most bindings provide a set of built-in channels automatically, so this is not always needed.                         |
+
+Channel Map:
+
+| Channel Configuration Key | Description                                                                                                                                                                          |
+|:--------------------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `<channel_id>`            | The channel ID. It is a single word that when combined with the Thing UID forms the full channel UID.                                                                                |
+| `type`                    | Channel type as defined by the Binding. When the channel type is defined, `kind`, `itemType`, and `itemDimension` should not be defined because the channel type will override them. |
+| `kind`                    | Whether it's a State channel or a Trigger channel. Default: `state`                                                                                                                  |
```

Comment:
"state" and "trigger" whatever their case are the unique valid values.

Reply:
I've changed the to `state` and `trigger` for better clarity.